internal/pyxis/pyxis.go

```diff
@@ -425,21 +429,30 @@ func (p *pyxisClient) createTestResults(ctx context.Context, testResults *TestRe
 
 	defer resp.Body.Close()
 
-	body, err := io.ReadAll(resp.Body)
+	respBody, err := io.ReadAll(resp.Body)
 	if err != nil {
 		return nil, fmt.Errorf("could not read body: %w", err)
 	}
 
+	body := string(respBody)
 	if ok := checkStatus(resp.StatusCode); !ok {
+		// checking to see if the users is using an unsupported versions
+		// if so display preflights latest download url to the user
+		if resp.StatusCode == http.StatusBadRequest &&
+			(strings.Contains(body, "not recognized") || strings.Contains(body, "not supported")) {
+			logger.Error(errors.New("invalid preflight version, please please download the latest version and re-submit"),
+				"release-url", fmt.Sprintf("https://%s/releases/latest", version.Version.Name))
+		}
+
 		return nil, fmt.Errorf(
-			"status code: %d: body: %s",
+			"status code: %d: respBody: %s",
 			resp.StatusCode,
-			string(body))
+			body)
 	}
 
 	newTestResults := TestResults{}
-	if err := json.Unmarshal(body, &newTestResults); err != nil {
-		return nil, fmt.Errorf("could not unmarshal body: %s: %w", string(body), err)
+	if err := json.Unmarshal(respBody, &newTestResults); err != nil {
+		return nil, fmt.Errorf("could not unmarshal body: %s: %w", body, err)
```

Comment:
I don't think we should be mixing these here... Use `body` for both or `respBody` for both.

Reply:
This is why just having `string(body)` everywhere is more clear then trying to have another variable IMO.